AutoscoperM/AutoscoperM.py

```diff
@@ -337,7 +337,8 @@ def onSampleDataButtonClicked(self, dataType):
 
         # Ensure that autoscoper is running
         if self.logic.AutoscoperProcess.state() not in [qt.QProcess.Starting, qt.QProcess.Running]:
-            logging.error("Autoscoper is not running. Please start Autoscoper using the AutoscoperM module.")
+            if slicer.util.confirmYesNoDisplay("Autoscoper is not running. Do you want to start Autoscoper?"):
+                self.lookupAndStartAutoscoper()
```

Comment:
Do you think it's worthwhile adding `self.onSampleDataButtonClicked(datatype)` below this line, so the sample data is automatically loaded in once autoscoper is launched? 

Other than that everything LGTM.

Reply:
It turns that this code is already added as part of `onSampleDataButtonClicked`

Reply:
That makes sense and thanks for the review/test/fix :pray: